ios/ButtonManager.m

```diff
@@ -19,4 +20,12 @@ @interface RCT_EXTERN_MODULE(ButtonManager, RCTViewManager)
 RCT_EXPORT_VIEW_PROPERTY(minLongPressDuration, NSTimeInterval)
 RCT_EXPORT_VIEW_PROPERTY(scaleTo, CGFloat)
 
-@end 
+@end
+
+@implementation ButtonManager : RCTViewManager
```

Comment:
what's the motivation of making it objC? Is there any reason but for a clean code? (asking from curiosity)

Reply:
No, just for the sake of having fewer files. Previously, we had 1 extra file per manager just because I wanted these 4 lines written in Swift: 
```objc
- (UIView *view)view {
  return ...;
}
```
After a while, I find it too verbose and pointless, so moved it back to the initial state (everything is in objc)